routes.js

```diff
@@ -29,38 +29,43 @@ module.exports = (app, logger) => {
   // The form API call
   app.post('/join', (req, res, next) => {
     // Github actions
-    logger.info(`Request body ${JSON.stringify(req.body)}`)
+    logger.info('Request body', req.body)
```

Comment:
It turns out that winston has a better, compact way to display objects than just stringifying them. It outputs like `key1=val1, key2=val2, ...` which looks nicer IMO than `{"key1":"val1","key2":"val2",...}`